network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -43,7 +43,7 @@
 
 
 {% block additional_head_elements %}
-  <meta name="monetization" content="$pay.stronghold.co/1a1938a9a3ec0ea4c539d779be2b7e1c9ee">
+  <meta name="monetization" content="$ilp.uphold.com/aKpM9kwnGBy4">
```

Comment:
can we make that identifier this an env var?

Reply:
I suppose that'd make it easier to update in the future, but I doubt it'll change again anytime soon.